scss/components/forms.scss

```diff
@@ -1295,7 +1389,33 @@
       }
     }
 
-    [_type="flStarRating"] .rating-input:checked~.rating-star .fa {
+    [_type="flStarRating"] .rating-input~.readonly.rating-star .fa {
+      &:before {
+        content: "\f005";
+        -webkit-text-stroke: 1px map-get($configuration, formReadOnlySelectedFillColor);
+      }
+
+      // Styles for tablet
+      @include above($tabletBreakpoint) {
+        content: "\f005";
```

Comment:
@NikitaJadhavXoriant There shouldn't be any property in theme SCSS files that do not use at least one variable from the theme.

If it's not using any variable like this line, it should go into the appropriate library/widget CSS file. In this case, it looks like it should go into the Form widget CSS files.

Please verify this for the entire PR.

Reply:
Done the changes accordingly and added in PR